### DISCUSSION TITLE
Solid mechanics library missing

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#27284] [Q&A Getting Started]
I discovered I do not have the solid mechanics library in my MOOSE repository.

**Comment by Emmanuel-324** [comment]
The mentioned library is missing my application

**Reply by GiudGiud** [reply]
Hello

the tensor mechanics module got renamed to solid mechanics a month or so ago

If you update moose you should see it

Guillaume

**Reply by Emmanuel-324** [reply]
I see tensor mechanics in my MAKEFILE. I just built a new application

**Reply by GiudGiud** [reply]
If you used a new-enough moose you should see solid_mechanics instead of tensor mechanics in the Makefile
